src/components/header-bottom-sheet.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { FC } from 'react';
+import { StyleSheet, View} from 'react-native';
+import { paperTheme } from '../theme/paper-theme';
+
+interface HeaderBottomSheetProps {
+    color?: string,
+    height?: number | string,
+    width?: number | string
+}
+export const HeaderBottomSheet: FC<HeaderBottomSheetProps> = (props) => {
+    const style = StyleSheet.create({
+        headerContainer: {
+            alignItems: 'center',
+            paddingTop: 15,
```

Comment:
Make the padding all around the "pullItem"
```suggestion
            padding: 15,
```